cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -90,6 +90,7 @@ message SearchSeismicsRequest {
     bool include_partition = 9; // If true, include info on the partition. Must be data manager.\
     bool include_coverage = 10; // Deprecated. Use `coverage` instead.
     CoverageSpec coverage = 11; // If specified, requests coverage as well.
+    bool purge_cache = 12; // If true, purges cached coverages and recalculates
```

Comment:
Naming is fun, ain't it? I wonder if something like `recompute_coverage` would be better.

Reply:
What's wrong with purge 😭

Reply:
Feels a bit technical to me.. I don't know.

Reply:
I guess it can feel a bit harsh in a non-technical context

Reply:
Oh, here's what I don't like about it: It's not quite what it indicates. I would expect purge-cache to remove the cached version, irrespective of whether we request coverage or not. The current implementation instead forces recomputation if we request coverage, but does nothing otherwise.

Reply:
To be clear, I think the current implementation is good. It's just that I think it doesn't exactly purge the cache (in all cases), but it does force recomputation.

Reply:
On naming: You don't have to _purge_ a cache, it's fine to invalidate it. You know, cache invalidation and naming things.

Reply:
Frankly, I'm... not happy that we expose this in the API this way, though. And it makes the SearchSeismics non-idempotent in a strict sense, ie it can mutate our internal state (even if the state is just a cache).

I would rather we associate this with ingestion somehow - maybe we should just clear out the coverage columns in the seismicstore during ingestion?

Reply:
Ok, in that case I'll merge my debug hack to skip the cache for now.

Reply:
Ok, in that case I'll merge my debug hack to skip the cache for now.

Reply:
@amorken It will only be non-idempotent if some logic is incorrect on our side. I think the design problems come in because caches are designed to be transparent to users, whereas APIs are all stuff that aren't transparent to users. 

Anyways, I will close this for now and go with the debug hack of audun's instead